basil/TL/Visa.py

```diff
@@ -45,7 +50,27 @@ def write(self, data):
         self._resource.write(data)
 
     def read(self):
-        self._resource.read()
+        if self._resource.read_termination == "":
+            ret = ""
+            while True:
```

Comment:
i guess this can cause an infinite loop if there are mismatches in termination definitions? maybe better to use a break based on timeout or number of loops here?